packages/gitbook-v2/src/lib/data/urls.test.ts

```diff
@@ -362,6 +362,28 @@ describe('getURLLookupAlternatives', () => {
             ],
         });
     });
+
+    it('should skip proxy root URLs', () => {
+        expect(
+            getURLLookupAlternatives(new URL('https://proxy.gitbook.site/site_123/doc1'))
```

Comment:
I think the real case is more something like `https://proxy.gitbook.site/sites/site_foo/hello/world`